tools/ubergen/bin/ubergen.ts

```diff
@@ -59,9 +59,14 @@ interface PackageJson {
   readonly name: string;
   readonly types: string;
   readonly version: string;
+  readonly stability: string;
   readonly [key: string]: unknown;
+  readonly 'cdk-build': {
+    readonly cloudformation: string[] | string;
+  };
   readonly ubergen?: {
     readonly deprecatedPackages?: readonly string[];
+    readonly stripExperimental?: boolean;
```

Comment:
'strip' is not the right verb here. Maybe 'exclude' or 'drop' would be better.

Reply:
Tried `excludeExperimentalL2`,  wdyt?

Reply:
excludeExperimentalModules